include/gz/gui/Application.hh

```diff
@@ -37,191 +37,188 @@ namespace tinyxml2
   class XMLElement;
 }
 
-namespace gz
+namespace gz::gui
```

Comment:
Do we plan to update all the libs within the gazebo ecosystem? If not, I'm slightly against making these changes so that we can keep all the libs in similar style

Reply:
Aspirationally, I would like to.  Realistically, it takes time.  I think that the more condensed looks and feels better, so moving in that direction over time seems appealing.

Reply:
Is there a reason we aren't starting at gz-gui7 or earlier? Will this make forward ports more painful?

Reply:
I think we need at least C++17, so targeting gui7 would be reasonable.

Reply:
let's do that. Also, we should create an issue that tracks the changes for all the other libs with a 'help wanted' label. It could be easy tasks for outside contributors

Reply:
Just opening another PR seems easier: https://github.com/gazebosim/gz-gui/pull/590